docs/src/atomistic/engines/ipi.rst

```diff
@@ -0,0 +1,69 @@
+.. _engine-ipi:
+
+i-PI
+====
+
+.. list-table::
+   :header-rows: 1
+
+   * - Official website
+     - How is metatensor supported?
+   * - https://ipi-code.org/
+     - In the official version
+
+
+Supported model outputs
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Only the :ref:`energy <energy-output>` output is supported, and used to run path
+integral simulations, incorporating quantum nuclear effects in the statistical
+sampling.
+
+How to install the code
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The metatensor interface is part of i-PI since version 3.0. Please refer to
+`i-PI documentation`_ about how to install it.
+
+.. _i-PI documentation: https://ipi-code.org/i-pi/getting-started.html#installing-i-pi
+
+How to use the code
+^^^^^^^^^^^^^^^^^^^
```

Comment:
I think a real example of running a simulation wit i-PI would be very cool. Giving the full input file etc.

Reply:
good point, I would leave this for a separate PR though (mainly because I don't know how to properly use i-PI myself)

Reply:
Yes, lets open as issue for this. Linking an explicit in example the ipi docs would also be fine.